include/lsst/afw/table/Schema.h

```diff
@@ -341,6 +341,19 @@ class SubSchema {
     typedef detail::SchemaImpl Impl;
 public:
 
+    //@{
+    /// Join strings using the field delimiter appropriate for this Schema
```

Comment:
Just because the other overloads predate our use of C++11, and I didn't want to change too much.